specs/pvs/corla.pvs

```diff
@@ -781,17 +828,18 @@ END cryptography
 
 %%% \subsection{Randomness}
 
-%%% High quality randomness is key to the legitimacy of any
-%%% risk-limiting audit.
+%%% Choosing a random sample of ballots is key to the legitimacy of any
+%%% risk-limiting audit. Legitimacy depends on a high-quality process for
+%%% the random samples. The Colorado RLA Tool will produce a random order 
+%%% of all the ballots once and for all; then random samples can be created 
```

Comment:
=> of all the ballots _for each county_ once and for all; then random samples _of the necessary size_ can be created